.gitignore

```diff
@@ -1,6 +1 @@
-*.sublime-*
-node_modules/
```

Comment:
It's a good idea to keep `node_modules` in your gitignore :)

Reply:
ah thanks!

Reply:
Okay, this should be fixed.